### PR TITLE
First pass at b10 docs. Modifications to other pages for intro and li…

### PR DESCRIPTION
**Pull request by richfirely**
…nks.

### CODE REVIEW

File: compliance/b_10.rst [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+This encompasses the ability to generate export files for individual patients and entire patient populations in a computable format. 
+Firely Server provides comprehensive support for this requirement through its :ref:`Bulk Data Export feature <feature_bulkdataexport>`.
+For a deeper understanding and comprehensive utilization of this feature beyond the (b)(10) requirements, 
+please consult the provided documentation, which includes setup instructions, application settings, parameters, filtering options, and additional details.
+
+File Format
+===========
+
+Bulk Data Export files are formatted in NDJSON (New line delimited JSON), a modified version of the JSON format designed for efficient bulk data transfer. 
+NDJSON follows a simplified JSON structure where resources are serialized without whitespace and separated by a newline pair (characters 13 and 10).
```
Comment (Sieuwkje-firely) [inline compliance/b_10.rst:17]: To me it is not clear what you mean with "(characters 13 and 10)".  Maybe adding e.g. ASCII characters here can already help

File: compliance/b_10.rst [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+please consult the provided documentation, which includes setup instructions, application settings, parameters, filtering options, and additional details.
+
+File Format
+===========
+
+Bulk Data Export files are formatted in NDJSON (New line delimited JSON), a modified version of the JSON format designed for efficient bulk data transfer. 
+NDJSON follows a simplified JSON structure where resources are serialized without whitespace and separated by a newline pair (characters 13 and 10). 
+This format aids in streamlining processing by ensuring that each NDJSON document contains only resources of a single type. 
+Each line within the document represents a resource of that specific type. 
+It is worth noting that while resources can include contained resources of various types, a single file can accommodate multiple resources for multiple patients with multiple records.
```
Comment (Sieuwkje-firely) [inline compliance/b_10.rst:20]: I guess this sentence confuses me a bit: 
An NDJSON document can only contain resources of one type (such as Patient, Observation etc.), the latter sentence of line 20  however almost seems to suggest that a single NDJSON file can contain multiple _different_ resource types. Could you clarify?

File: compliance/b_10.rst [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+Single Patient EHI Export - 170.315(b)(10)(i)
+=============================================
+
+To meet the requirements of 170.315(b)(10)(i), Firely Server offers an export feature that allows for the extraction of data related to a single patient. 
+This is achieved by utilizing a Parameters Resource filter against the Patient endpoint.
+
+However, an alternative method is also provided using the Group endpoint. 
+This alternative approach caters to real-world scenarios and alternative workflows which may be more suitable to portals and UI that connect to Firely Server.
+
+Notice in the following Parameters Reource example; there is just a single patient reference.
```
Comment (Sieuwkje-firely) [inline compliance/b_10.rst:57]: typo (Reource)

File: compliance/b_10.rst [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+This option exports all of the patients within the specified ``[groupId]``.
+
+3.2. Exporting with filtering parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Perform a POST request to the Group endpoint, including a Parameters filter in the request body. 
+
+1. Create a POST request to the Group endpoint:
+
+        ``POST {{BASE_URL}}/Group/$export``
```
Comment (Sieuwkje-firely) [inline compliance/b_10.rst:180]: Is it correct that you did not include a groupid in this request?

File: compliance/b_10.rst (hunk 2 above)
Comment (richfirely) [reply compliance/b_10.rst:20]: I'm stuck on this. Do you have a suggestion? In context of the other sentences it seems clear to me.

File: compliance/b_10.rst (hunk 4 above)
Comment (richfirely) [reply compliance/b_10.rst:180]: no. fixed

**Review by alexzautke — APPROVED** [review]

**Pull request merged**